### DISCUSSION TITLE
Good project

### DISCUSSION THREAD

**Discussion by ikay06** [shardeum/shardeum#144] [General]
Good project shardeum

**Comment by unyingsan** [comment]
why

**Comment by ototech7** [comment]
Thanks for starting the conversation and sharing your thoughts, @ikay06! I agree, Shardeum is an exciting project with its EVM based autoscaling architecture and dynamic state sharding, really pushing forward scalability and decentralization in the blockchain space. If you or anyone else has questions about running validator nodes, setting up a local dev environment, or troubleshooting common issues, feel free to reach out here, happy to lend a hand based on my own experience with multi-chain deployments and node operations. Great to see the community growing!
